### PR TITLE
Add `degree_subject_autocomplete` feature flag

### PR DESCRIPTION
**Pull request by martyn-w**
### Trello card
[Create feature flag for degree subject autocomplete](https://trello.com/c/nSPjIRdF/6234-create-feature-flag-for-degree-subject-autocomplete-work-get-school-experience)

### Context
The new degree subject autocomplete requires a feature flag so that it can be enabled/disabled quickly

### Changes proposed in this pull request
Add a new feature flag

### CODE REVIEW

File: .env.production [hunk 1]
```diff
@@ -8,3 +8,5 @@ DFE_SIGNIN_API_SCHOOL_CHANGE_ENABLED="1"
 GTM_ID=GTM-W3VGWP6
 GTM_UA_ID=UA-136255210-1
 BIGQUERY_DATASET=gse_events_production
+
+DEGREE_SUBJECT_AUTOCOMPLETE=0
```
Comment (asatwal) [inline .env.production:12]: Just for consistency with the flags above, should this be called `DEGREE_SUBJECT_AUTOCOMPLETE_ENABLED` with a value of "0" or "1"

**Review by asatwal — APPROVED** [review]
👍 - A minor comment just for consistency with other Boolean flags.

File: .env.production (hunk 1 above)
Comment (martyn-w) [reply .env.production:12]: yes, good point - I'll rename it - thanks 👍

Comment (martyn-w) [reply .env.production]: @asatwal now renamed to `DEGREE_SUBJECT_AUTOCOMPLETE_ENABLED`

**Pull request merged**